### PR TITLE
Exo fixes pairing

### PR DESCRIPTION
**Pull request by eldoniel**
This PR fixes the following issue:

https://github.com/claroline/Distribution/issues/803

Reseted proposals couldn't be moved again. This bug gets fixed.

### CODE REVIEW

**Comment by kitan1982** [comment]
LGTM

**Comment by stefk** [comment]
@eldoniel, can you point to the parts that actually changed? The fix is kinda lost in cs changes.

File: plugin/exo/Resources/modules/question/Controllers/Type/MatchQuestionCtrl.js [hunk 1]
```diff
@@ -450,130 +451,130 @@ MatchQuestionCtrl.prototype.onFeedbackHide = function onFeedbackHide() {
...
-};
+  if (this.question.toBind) {
+    jsPlumb.detachEveryConnection()
+    this.connections.splice(0, this.connections.length)
+  } else {
+    // init all proposals ui
+    $('.origin').each(function () {
+      if ($(this).find('.draggable').attr('style')) {
+        $(this).find('.draggable').removeAttr('style')
+      }
```
Comment (eldoniel) [inline plugin/exo/Resources/modules/question/Controllers/Type/MatchQuestionCtrl.js]: @stefk Here is done the bugfix. The two lines below were in the if above.

File: plugin/exo/Resources/modules/question/Controllers/Type/MatchQuestionCtrl.js [hunk 2]
```diff
@@ -450,130 +451,130 @@ MatchQuestionCtrl.prototype.onFeedbackHide = function onFeedbackHide() {
...
+      $(this).removeAttr('aria-disabled')
+      $(this).draggable('enable')
```
Comment (eldoniel) [inline plugin/exo/Resources/modules/question/Controllers/Type/MatchQuestionCtrl.js]: @stefk Here is done the bugfix. The two lines "removeAttr" and "draggable" were in the if above.

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**